### PR TITLE
Add Controlled Frame test app

### PR DESCRIPTION
**Pull request by odejesush**
This change adds the Controlled Frame test app to the test_app/ directory. The README contained within the directory explains the steps necessary to launch the app as an Isolated Web App with the <controlledframe> tag enabled.

### CODE REVIEW

File: test_app/manifest.webmanifest [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "IWA WebView Test",
```
Comment (chasephillips) [inline test_app/manifest.webmanifest:2]: Let's call this "IWA Controlled Frame Test"

File: test_app/manifest.webmanifest [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  "short_name": "IWAWVT",
```
Comment (chasephillips) [inline test_app/manifest.webmanifest:3]: IWACFTest

File: test_app/manifest.webmanifest [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  "description": "Isolated app to WebView.",
```
Comment (chasephillips) [inline test_app/manifest.webmanifest:4]: s/WebView/Controlled Frame/

File: test_app/index.html [hunk 4]
```diff
@@ -0,0 +1,633 @@
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+  <title>ControlledFrame in Isolated PWA Test</title>
```
Comment (chasephillips) [inline test_app/index.html:7]: s/ControlledFrame/Controlled Frame/

also, s/Isolated PWA/IWA/

File: test_app/index.html [hunk 5]
```diff
@@ -0,0 +1,633 @@
...
+
+  <link rel="manifest" href="/manifest.webmanifest" />
+  <link rel="stylesheet" href="/style.css" />
+</head>
+
+<body>
+  <div class="content">
+    <div class="grid">
+      <h1 class="heading">Isolated Web App ControlledFrame Test</h1>
```
Comment (chasephillips) [inline test_app/index.html:16]: s/ControlledFrame/Controlled Frame/

**Comment by odejesush** [comment]
Thanks for reviewing!

File: test_app/manifest.webmanifest [hunk 6]
```diff
@@ -13,6 +13,5 @@
   "display": "fullscreen",
   "theme_color": "white",
   "background_color": "white",
-  "isolated_storage": true,
-  "permissions_policy": { "direct-sockets": [ "self" ] }
```
Comment (chasephillips) [inline test_app/manifest.webmanifest]: +1

**Review by chasephillips — COMMENTED** [review]
LGTM, rebase+merged

**Pull request merged**